static/user.html

```diff
@@ -121,11 +121,20 @@ <h3>Danger zone</h3>
         const userUrl = '/api/users/' + urlEncodedUsername
         HTTP.request('GET', userUrl)
           .then(item => {
-            const hasPassword = item.password_hash.length ? '●' : '○'
+            const hasPassword = item.password_hash ? '●' : '○'
             document.getElementById('tags').textContent = item.tags
             document.getElementById('hasPassword').textContent = hasPassword
+            tagHelper(item.tags)
           }).catch(HTTP.standardErrorHandler)
       }
+
+      function tagHelper(tags) {
+        const vals = tags.split(',')
+        vals.forEach((val) =>  {
```

Comment:
should we have the same check here as in `argumentHelper` in `helpers.js`? 

```js
if (currentVal.includes(val)) { return }
```

Reply:
Difference is, that `argumentHelper` triggers on onclick event and only contains one tag (the one pressed in the UI). While `tagHelper` format all tags received from the API response, when displaying it in the input field.

Reply:
The check is used to avoid adding more of the same tag. Perhaps this should be handled by the API too. As for now it's possible to assign more than one of the same tag to a user.